ts/Core/Chart/GanttChart.ts

```diff
@@ -43,8 +37,129 @@ const {
 
 import '../../Series/Gantt/GanttSeries.js';
 
+class GanttChart extends Chart {
+    /**
+     * Initializes the chart. The constructor's arguments are passed on
+     * directly.
+     *
+     * @function Highcharts.GanttChart#init
+     *
+     * @param {Highcharts.Options} userOptions
+     *        Custom options.
+     *
+     * @param {Function} [callback]
+     *        Function to run when the chart has loaded and and all external
+     *        images are loaded.
+     *
+     * @return {void}
```

Comment:
If these should be public classes and functions, we need on top of the class pattern some doclets similar to
```ts
/**
 * Gantt-optimized chart. Use {@link Highcharts.Chart|Chart} for common charts.
 *
 * @requires modules/gantt
 *
 * @class
 * @name Highcharts.GanttChart
 * @extends Highcharts.Chart
 */
```

Otherwise the class doclets lack a `@private` tag one line above `@function`.